fabfile/setup.py

```diff
@@ -25,11 +25,8 @@ def create_project():
     env.settings = get_fab_settings()
     env.proj_name = get_project_name_from_repo(env.settings.get('git_repo'))
     env.settings['project_name'] = env.proj_name
-
-    with bash():
-        with lcd(env.proj_name):
-            local("cp -rf $FAB_PATH/../django_project/* .")
-        recursive_file_modify(os.path.abspath("./{0}".format(env.proj_name)), env.settings)
+    local("cp -rf ./fabfile/../django_project/ ./{0}".format(env.proj_name))
+    recursive_file_modify(os.path.abspath("./{0}".format(env.proj_name)), env.settings)
```

Comment:
using `{0}` is a bit unnecessary in a lot of these formats, especially since we're only using one input mostly